runtime/context.go

```diff
@@ -72,8 +87,8 @@ func AnnotateContext(ctx context.Context, mux *ServeMux, req *http.Request, rpcM
 
 // AnnotateIncomingContext adds context information such as metadata from the request.
 // Attach metadata as incoming context.
-func AnnotateIncomingContext(ctx context.Context, mux *ServeMux, req *http.Request, rpcMethodName string) (context.Context, error) {
-	ctx, md, err := annotateContext(ctx, mux, req, rpcMethodName)
+func AnnotateIncomingContext(ctx context.Context, mux *ServeMux, req *http.Request, rpcMethodName string, options ...AnnotateOption) (context.Context, error) {
```

Comment:
If any concern about interface compatibility, I'll define another function like below.
```
func AnnotateIncomingContextWithOption(ctx context.Context, mux *ServeMux, req *http.Request, rpcMethodName string, options ...AnnotateOption) (context.Context, error) {
```

Reply:
I think this is technically a breaking change because someone could be saving the old function signature as a variable and it would change type, but that should be a vanishingly rare use case so we can probably just add this variadic parameter without practically breaking anyone.